docs/tctl/cluster/get-search-attributes.md

```diff
@@ -8,7 +8,7 @@ tags:
   - tctl
 ---
 
-The `tctl cluster get-search-attributes` command lists all [Search Attributes](/concepts/what-is-a-search-attribute) that can be used in the `--query` modifier of the [`tctl workflow list`](/tctl/workflow/list) command.
+The `tctl cluster get-search-attributes` command lists all [Search Attributes](/concepts/what-is-a-search-attribute) that can be used in the `--query` modifier of the [`tctl workflow list`](/tctl/workflow/list) command and the `--search_attr_key` and `--search-attr_value` modifiers of the [`tctl workflow run`](/tctl/workflow/run) and [`tctl workflow start`](/tctl/workflow/start) commands.
```

Comment:
I haven't gotten this to work myself yet (temporalio/tctl#292), but the docs in `tctl` show this as `--search_attr_value` instead of `--search-attr_value`.

Reply:
_D'oh._ That's what I get for typing it instead of copying and pasting. Thanks for the catch!